ui/components/MesheryPatterns.js

```diff
@@ -1503,6 +1503,10 @@ function MesheryPatterns({
       { credentials: 'include', method: 'POST', body: requestBody },
       () => {
         updateProgress({ showProgress: false });
+        notify({
+          message: `${name} Design Uploaded`,
```

Comment:
Please put name in quotes.
Lowercase "design" and "uploaded". This keeps the notifications consistent (see above).